homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -135,10 +141,10 @@ async def async_setup_entry(hass: HomeAssistant, config: ConfigEntry) -> bool:
     hass.data[DOMAIN][config.entry_id] = SwitchbotCloudData(
         api=api, devices=make_device_data(hass, api, devices, coordinators_by_id)
     )
-    await hass.config_entries.async_forward_entry_setups(config, PLATFORMS)
     await gather(
         *[coordinator.async_refresh() for coordinator in coordinators_by_id.values()]
     )
+    await hass.config_entries.async_forward_entry_setups(config, PLATFORMS)
```

Comment:
This change is correct, but we should do this in a separate PR

Reply:
PR opened: #135625